po/es.po

```diff
@@ -186,356 +194,742 @@ msgstr "marcado como desactualizado: "
+"del sistema"
+
+#: src/help.rs:69
+msgid ""
+"    --[no]combinedupgrade  Refresh then perform the repo and AUR upgrade "
+"together"
+msgstr ""
+"    --[no]combinedupgrade  Refrescar antes de ejecutar la actualización de "
+"repositorios y AUR simultáneamente"
+
+#: src/help.rs:70
+msgid ""
+"    --[no]batchinstall     Build multiple AUR packages then install them "
+"together"
+msgstr ""
+"    --[no]batchinstall     Construir multiples paquetes AUR e instalarlos a "
+"la vez"
+
+#: src/help.rs:71
+msgid ""
+"    --[no]provides         Look for matching providers when searching for "
+"packages"
+msgstr ""
+"    --[no]provides         Buscar proveedores que coincidan al buscar "
+"paquetes"
+
+#: src/help.rs:72
+msgid "    --[no]devel            Check development packages during sysupgrade"
+msgstr ""
+"    --[no]devel            Comprobar paquetes de desarrollo durante la "
+"actualización del sistema"
+
+#: src/help.rs:73
+msgid ""
+"    --[no]installdebug     Also install debug packages when a package "
+"provides them"
+msgstr ""
+"    --[no]installdebug     Instalar también paquetes de debug cuando un "
+"paquete lo provee"
+
+#: src/help.rs:74
+msgid ""
+"    --[no]sudoloop         Loop sudo calls in the background to avoid timeout"
+msgstr ""
+"    --[no]sudoloop         Realizar llamadas a sudo en segundo plano para "
+"evitar timeout"
+
+#: src/help.rs:75
+msgid "    --[no]chroot           Build packages in a chroot"
+msgstr "    --[no]chroot           Construir paquetes en un chroot"
+
+#: src/help.rs:76
+msgid "    --[no]sign             Sign packages with gpg"
+msgstr "    --[no]sign             Firmar paquetes con gpg"
+
+#: src/help.rs:77
+msgid "    --[no]signdb           Sign databases with gpg"
+msgstr "    --[no]signdb           Firmar bases de datos con gpg"
+
+#: src/help.rs:78
+msgid "    --localrepo            Build packages into a local repo"
+msgstr "    --localrepo            Construir paquetes en un repositorio local"
+
+#: src/help.rs:79
+msgid ""
+"    --nocheck              Don't resolve checkdepends or run the check "
+"function"
+msgstr ""
+"    --nocheck              No resolver checkdepends o ejecutar función de "
+"comprobación"
+
+#: src/help.rs:80
+msgid ""
+"    --develsuffixes        Suffixes used to decide if a package is a devel "
+"package"
+msgstr ""
+"    --develsuffixes        Sufijos utilizados para decidir si un paquete es "
+"un paquete de desarrollo"
+
+#: src/help.rs:81
+msgid ""
+"    --bottomup             Shows AUR's packages first and then repository's"
+msgstr ""
+"    --bottomup             Muestra primero los paquetes de AUR y después los "
+"de los repositorios"
+
+#: src/help.rs:82
+msgid ""
+"    --topdown              Shows repository's packages first and then AUR's"
+msgstr ""
+"    --topdown              Muestra primero los paquetes de los repositorios "
+"y después de los AUR"
+
+#: src/help.rs:84
+msgid "show specific options:"
+msgstr "mostrar opciones específicas:"
```

Comment:
show is the name of the argument: `--show`

Reply:
Updated to mean something more alike to that. Thanks for checking it, without context I couldn't know.

Reply:
It's changed but --show still seems to be translated.

Reply:
You want "show" not to be translated? Now it means something like "specific options of view" so it's clear it's refering to the specific options of the looks, but I can leave it as "show" without translation if you prefer

Reply:
It makes more sense in the context of the help menu 

```
New operations:
    paru {-P --show}        [options]
    paru {-G --getpkgbuild} [package(s)]

show specific options:
    -c --complete         Used for completions
    -s --stats            Display system package statistics
    -w --news             Print arch news

getpkgbuild specific options:
    -p --print            Print pkgbuild to stdout
    -c --comments         Print AUR comments for pkgbuild

upgrade specific options:
    -i --install          Install package as well as building
```

Reply:
That should be better now, I also changed the "upgrade specific options" in the same way